### PR TITLE
To be consistent with V1 production

### PR DESCRIPTION
**Pull request by Zhiliqiao888**

### CODE REVIEW

**Comment by Zhiliqiao888** [comment]
@Lauragra  Can we have someone to start review? Thank you.

File: api-reference/v1.0/api/reportroot-getteamsdeviceusageuserdetail.md [hunk 1]
```diff
@@ -114,7 +117,7 @@ Follow the 302 redirection and the CSV file that downloads will have the followi
 HTTP/1.1 200 OK
 Content-Type: application/octet-stream
 
-Report Refresh Date,User Id,User Principal Name,Last Activity Date,Is Deleted,Deleted Date,Used Web,Used Windows Phone,Used iOS,Used Mac,Used Android Phone,Used Windows,Report Period
+Report Refresh Date,User Id,User Principal Name,Last Activity Date,Is Deleted,Deleted Date,Used Web,Used Windows Phone,Used iOS,Used Mac,Used Android Phone,Used Windows,Used  Chrome OS,Used Linux,Is Licensed,Report Period
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/reportroot-getteamsdeviceusageuserdetail.md:120]: For consistency, please remove the one additional empty space between Used and Chrome => 

```
...Used Windows,Used Chrome OS,Used Linux...
```

File: api-reference/v1.0/api/reportroot-getteamsuseractivitycounts.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "reportRoot: getTeamsUserActivityCounts"
-description: "Get the number of Microsoft Teams activities by activity type. The activity types are team chat messages, private chat messages, calls, and meetings."
+description: "Get the number of Microsoft Teams activities by activity type. The activities are performed by Microsoft Teams licensed users."
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/reportroot-getteamsuseractivitycounts.md:3]: Please, update the corresponding method description also [here](https://docs.microsoft.com/en-us/graph/api/resources/microsoft-teams-user-activity-reports?view=graph-rest-1.0).

File: api-reference/v1.0/api/reportroot-getteamsdeviceusageusercounts.md [hunk 3]
```diff
@@ -62,9 +62,13 @@ The CSV file has the following headers for columns:
 - iOS
 - Mac
 - Windows
+- Chrome OS
+- Linux
 - Report Date
 - Report Period
 
+**Note**: Column Windows Phone is deprecated and value will always be 0. It is kept in response just for backward compatibility purpose.
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/reportroot-getteamsdeviceusageusercounts.md:70]: For consistency, please replace Note with [!CAUTION]. Suggested revision:

```
> [!CAUTION] 
> The **Windows Phone** column is deprecated and its value will always be `0`. It's kept in the response for backward compatibility reasons.
```

Please, apply this change also here:

- api-reference/v1.0/api/reportroot-getteamsdeviceusagedistributionusercounts.md

File: changelog/Microsoft.O365Reporting.json [hunk 4]
```diff
@@ -1,5 +1,55 @@
 {
   "changelog": [
+	{
+		"ChangeList": [
+		 {
+			"Id": "b1d5ed13-e5db-42d0-88fe-7bae2ec7e18c",
+			"ApiChange": "Method",
+			"ChangedApiName": "getTeamsUserActivityCounts",
+			"ChangeType": "Addition",
+			"Description": "Added support for the [getTeamsUserActivityCounts](https://docs.microsoft.com/en-us/graph/api/reportroot-getTeamsUserActivityCounts?view=graph-rest-1.0) method.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.O365Reporting.json:10]: Please, update this description to include the columns that were added to this method. Suggested revision:

- Added the \*\*Post Messages\*\*, \*\*Reply Messages\*\*, \*\*Audio Duration\*\*, \*\*Video Duration\*\*, \*\*Screen Share Duration\*\*, \*\*Meetings Organized\*\*, \*\*Meetings Attended\*\* columns to the [getTeamsUserActivityCounts](...) method.

Please, update the description of the other changelog entries to contain the columns added.

File: changelog/Microsoft.O365Reporting.json [hunk 5]
```diff
@@ -1,5 +1,55 @@
 {
   "changelog": [
```
Comment (JarbasHorst) [inline changelog/Microsoft.O365Reporting.json:2]: For transparency and consistency, please include two new entries to this changelog record containing the columns deprecated. Suggested revision:

```
{
   "Id": "b1d5ed13-e5db-42d0-88fe-7bae2ec7e18c",
   "ApiChange": "Method",
   "ChangedApiName": "getTeamsDeviceUsageDistributionUserCounts",
   "ChangeType": "Deprecation",
   "Description": "Deprecated the **Windows Phone** column on the [getTeamsDeviceUsageDistributionUserCounts](https://docs.microsoft.com/en-us/graph/api/reportroot-getTeamsDeviceUsageDistributionUserCounts?view=graph-rest-1.0) method.",
   "Target": "getTeamsDeviceUsageDistributionUserCounts"
}
```

**Same for the getTeamsDeviceUsageUserCounts topic.**

**Comment by JarbasHorst** [comment]
Hello @Zhiliqiao888, I reviewed your PR and left some comments. Please, take a look when you will have time.

File: api-reference/v1.0/api/reportroot-getteamsdeviceusageusercounts.md (hunk 3 above)
Comment (Zhiliqiao888) [reply api-reference/v1.0/api/reportroot-getteamsdeviceusageusercounts.md:70]: Done

File: changelog/Microsoft.O365Reporting.json (hunk 4 above)
Comment (Zhiliqiao888) [reply changelog/Microsoft.O365Reporting.json:10]: Updated

**Comment by Zhiliqiao888** [comment]
> Hello @Zhiliqiao888, I reviewed your PR and left some comments. Please, take a look when you will have time.

Jarbas Thank you. Could you re-review for the updated parts? @JarbasHorst

File: api-reference/v1.0/api/reportroot-getteamsuseractivitycounts.md (hunk 2 above)
Comment (Zhiliqiao888) [reply api-reference/v1.0/api/reportroot-getteamsuseractivitycounts.md:3]: Updated

**Comment by JarbasHorst** [comment]
@Zhiliqiao888, please, merge PRs [15](https://github.com/Zhiliqiao888/microsoft-graph-docs/pull/15) and [16](https://github.com/Zhiliqiao888/microsoft-graph-docs/pull/16) into this branch to include my final edits.

@Lauragra this PR is ready for final review after the author merged my changes.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**